Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/AttributesSectionTranslator.swift

```diff
@@ -64,13 +63,21 @@ struct AttributesSectionTranslator: RenderSectionTranslator {
                         return RenderAttribute.allowedTypes(tokens)
                     case (.allowedValues, let values as [SymbolGraph.AnyScalar]):
                         let stringValues = values.map { String($0) }
-                        return RenderAttribute.allowedValues(stringValues)
+                        if symbol.possibleValuesSectionVariants.allValues.isEmpty {
+                            return RenderAttribute.allowedValues(stringValues)
+                        }
+                        return nil
```

Comment:
very minor: This is computing the string values even when they're not used. You could move it into the if statement but since the entire case has the same requirement you could also move the condition up to the case itself. 

```diff
- case (.allowedValues, let values as [SymbolGraph.AnyScalar]):
+ case (.allowedValues, let values as [SymbolGraph.AnyScalar]) where !symbol.possibleValuesSectionVariants.allValues.isEmpty:
      let stringValues = values.map { String($0) }
      return RenderAttribute.allowedValues(stringValues)
```

Reply:
I realised we don't need the if anymore. The check was to maintain old behaviour, but since the only symbols that define possible values are OAS related symbols IMO is ok to just introduce this new logic for every case.

Reply:
Now that we are rendering a separate section just for possible values we don't need to show this values under the attributes section anymore, so this logic can be removed.